Main/Tests.cpp

```diff
@@ -145,17 +146,63 @@ bool TestRunner::executeTest(const std::wstring& dir, const std::wstring& testNa
 
 	ArmipsArguments args;
 	StringList errors;
+	int expectedRetVal = 0;
+	int retVal = 0;
+	bool checkRetVal = false;
+	bool result = true;
+
+	if (fileExists(L"commandLine.txt"))
```

Comment:
Possible, but since the text file also includes the return value to check for, which is not a parameter/argument, I felt `commandLine.txt` was more fitting here. But if you prefer `parameters.txt`, let me know and I'll go ahead and change it.